Formula/genext2fs.rb

```diff
@@ -0,0 +1,15 @@
+class Genext2fs < Formula
+  desc "Generates an ext2 filesystem as a normal (non-root) user"
+  homepage "http://genext2fs.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/genext2fs/genext2fs/1.4.1/genext2fs-1.4.1.tar.gz"
+  sha256 "404dbbfa7a86a6c3de8225c8da254d026b17fd288e05cec4df2cc7e1f4feecfc"
+
+  def install
+    system "./configure", "--disable-debug", "--disable-dependency-tracking", "--disable-silent-rules", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/genext2fs", "--root", testpath, "--size-in-blocks", "20", File.join(testpath, 'test.img')
```

Comment:
File.join - is true way (in Ruby) to build path. "#{testpath}/test.img" - is dirty way. 
It is not dancing, please, read this: 
http://stackoverflow.com/questions/4114480/when-is-file-join-useful
http://stackoverflow.com/questions/597488/how-to-do-a-safe-join-pathname-in-ruby

Yep, line is too long anyway